### PR TITLE
Fix the ordered list discrepancy on getting started page

### LINKED ISSUE #653: [📝 Docs]: Fix documentation anomalies [UI/UX change]
### What Docs changes are you proposing?

on the page [getting-started-step-by-step](https://json-schema.org/learn/getting-started-step-by-step) we see a lot of anomalies in the numbering. For example - 

image

and 

image

CC - @benjagm  @gregsdennis 
Do let me know if they seem valid to  be fixes and I can go ahead and create a PR for the same?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by bhavukkalra**
**What kind of change does this PR introduce?**
Fixes the ordered list Discrepancy observed on the [getting-started-step-by-step](https://json-schema.org/learn/getting-started-step-by-step#optional) as reported in the issue #653 

**Issue Number:**
-  Closes #653 

**Screenshots of fix**
image

image

**Summary**
It was a markdown spacing issue. Found the fix here [markdown-continue-numbered-list](https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list) and applied. 

`Note` - This might need to be applied to other pages as well which has this problem.

**Does this PR introduce a breaking change?**
No

CC - @benjagm @gregsdennis  @jviotti

### CODE REVIEW

File: pages/learn/getting-started-step-by-step.md [hunk 1]
```diff
@@ -98,33 +98,31 @@ Using the product catalog example, `productId` is a numeric value that uniquely
 
 To add the `properties` object to the schema:
 
-1. Add the `properties` validation keyword to the end of the schema:
+1. Add the `properties` validation keyword to the end of the (test1)schema:
```
Comment (jviotti) [inline pages/learn/getting-started-step-by-step.md:101]: Is this change intended?

Comment (bhavukkalra) [reply pages/learn/getting-started-step-by-step.md:101]: Oh, that's a typo. Removing it.

**Review by benjagm — APPROVED** [review]
It worked. Thanks a lot!!

File: pages/learn/getting-started-step-by-step.md [hunk 2]
```diff
@@ -176,43 +174,40 @@ To define a required property:
...
-  }
+    ```jsonc
+      "properties": {
+        ...
+        "price": {
+          "description": "The price of the product",
+          "type": "number"
+        }
+      }
 ```
```
Comment (notEthan) [inline pages/learn/getting-started-step-by-step.md:185]: this one missed indentation

File: pages/learn/getting-started-step-by-step.md [hunk 3]
```diff
@@ -363,68 +354,63 @@ To create a nested data structure:
...
-...
-  "dimensions": {
-    "type": "object",
-  }
+    ```jsonc
+    ...
+      "dimensions": {
+        "type": "object",
+      }
 ```
```
Comment (notEthan) [inline pages/learn/getting-started-step-by-step.md:372]: here too

File: pages/learn/getting-started-step-by-step.md [hunk 4]
```diff
@@ -363,68 +354,63 @@ To create a nested data structure:
...
+            "type": "number"
+          },
+          "height": {
+            "type": "number"
+          }
+        }
       }
-    }
-  }
 ```
```
Comment (notEthan) [inline pages/learn/getting-started-step-by-step.md:392]: here too

File: pages/learn/getting-started-step-by-step.md (hunk 2 above)
Comment (benjagm) [reply pages/learn/getting-started-step-by-step.md:185]: Thanks for finding this @notEthan !! 

@bhavukkalra can you create a new PR to fix this? I merged it too fast.

File: pages/learn/getting-started-step-by-step.md (hunk 4 above)
Comment (benjagm) [reply pages/learn/getting-started-step-by-step.md:392]: Huge thanks for finding this!

Bhavuk can you create a new PR to fix this? I merged it too fast.
